alacritty/src/event.rs

```diff
@@ -988,6 +988,69 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
         self.update_search();
     }
 
+    #[inline]
+    fn search_region(&mut self, direction: Direction) {
+        let term = self.terminal();
+        let cursor_point = term.vi_mode_cursor.point;
+
+        let region: Option<String> = match term.selection {
+            Some(_) => {
+                let text = term.selection_to_string().filter(|s| !s.is_empty());
+                self.clear_selection();
+                text
+            },
```

Comment:
Does this require vi mode? I feel like in theory it should work just fine outside of it. The selection itself can be used as origin in both modes.

Reply:
could you elaborate?